File: src/main/java/com/harleylizard/magic_things/common/BiomeSetter.java
package com.harleylizard.magic_things.common;

import net.minecraft.core.Holder;
import net.minecraft.world.level.biome.Biome;
import net.minecraft.world.level.chunk.PalettedContainer;

public interface BiomeSetter {

    void magicThings$set(PalettedContainer<Holder<Biome>> biomes);

}

File: src/main/kotlin/com/harleylizard/magic_things/common/BiomeSetter.kt
package com.harleylizard.magic_things.common

import net.minecraft.core.Holder
import net.minecraft.world.level.biome.Biome
import net.minecraft.world.level.chunk.PalettedContainer

interface BiomeSetter {

    fun `magicThings$set`(biomes: PalettedContainer<Holder<Biome>>)

}